jiant/tasks/lib/templates/squad_style/core.py

```diff
@@ -21,6 +21,9 @@
 
 import logging
 
+# Store the tokenizers which insert 2 separators tokens
+MULTI_SEP_TOKENS_TOKENIZERS_SET = {"roberta", "camembert", "bart"}
```

Comment:
Thoughts on moving tokenizer specific wrapping constants to a dedicated file (ext/tokenizers/constants.py or similar)? Worried about the logic being scattered throughout files.

Reply:
This constant comes from Transformers: https://github.com/huggingface/transformers/blob/969859d5f67c7106de4d1098c4891c9b03694bbe/src/transformers/data/processors/squad.py#L17

Unfortunately, it comes in a later version than what we're currently requiring, so we can't currently import it (unless we bump the version, I'd like to wait until after Fall to do it because of the v3.0.2 issue). The upside is that it's from the same file that all the rest of the SQuAD code comes from, so it'll be consistent for anyone looking up update the SQuAD implementation in the future.